app/views/shared/check_answers/_vehicles.html.erb

```diff
@@ -40,22 +36,17 @@
       <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__vehicle__#{index}_change_link" }) do |row| %>
         <%= row.with_key(text: t(".providers.vehicle_index", sequence: index + 1), classes: "govuk-!-width-one-half govuk-heading-m") %>
         <%= row.with_value(text: nil) %>
-            <%= row.with_action(
-                  text: t("generic.change"),
-                  href: providers_legal_aid_application_means_vehicle_detail_path(@legal_aid_application, vehicle),
-                  visually_hidden_text: "vehicle #{index + 1}",
-                  ) %>
         <%= row.with_action(
               text: t("generic.change"),
               href: providers_legal_aid_application_means_vehicle_detail_path(@legal_aid_application, vehicle),
-              visually_hidden_text: t(".providers.vehicle_index", sequence: index + 1).downcase,
+              visually_hidden_text: "vehicle #{index + 1}",
             ) %>
       <% end %>
 
       <% if @legal_aid_application.applicant.has_partner_with_no_contrary_interest? %>
         <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__vehicles_owner" }) do |row| %>
           <%= row.with_key(text: t(".#{journey_type}.owner"), classes: "govuk-!-width-one-half") %>
-          <%= row.with_value { @vehicle_owner } %>
+          <%= row.with_value { t(".providers.options.#{vehicle.owner}") } %>
```

Comment:
~~not a blocker, especially as it is done elsewhere and Ihave little context on what this is displaying, but dynamic translations are untestable/specable by i18n-task testing gem (memories of CCCD come flooding back.~~

🤔 but you are cleaning up an equally dynamic translation in the controller i see, so np.

Reply:
We have a lot of dynamically built translations and I too get CCCD flashbacks 😄